test/system/ssh_keys_test.go

```diff
@@ -78,8 +78,8 @@ spec:
       terminationGracePeriodSeconds: 1
       containers:
       - name: test-ssh-keys
-        image: ubuntu
-        command: ["bash"]
+        image: quay.io/kinvolk/nginx:latest-amd64
```

Comment:
I'd rather use official images when possible.

Reply:
I couldn't find nginx official image on quay. Should we use binami one?

Reply:
Yeah, `bitnami/nginx` should good to me.